services/catarse/spec/models/user_spec.rb

```diff
@@ -168,6 +168,11 @@
   end
 
   describe '.find_active!' do
+    it 'should raise error when user is banned' do
```

Comment:
Acho que `it 'raises an error when user is banned' do...` seria melhor :thinking: